src/components/common/nested-select.vue

```diff
@@ -2,6 +2,7 @@
   <v-select
     class="request-action-select"
     filled
+    hide-details="auto"
```

Comment:
This keeps the component "short" unless some detail messages (errors) need to be shown. This is to prevent excess bottom whitespace, which we sometimes absorb with negative bottom margins -- but this is better.